prov/efa/src/rxr/rxr_init.c

```diff
@@ -91,6 +91,12 @@ static void rxr_init_env(void)
 	};
 
 	fi_param_get_int(&rxr_prov, "tx_min_credits", &rxr_env.tx_min_credits);
+	if (rxr_env.tx_min_credits <= 0) {
+		fprintf(stderr,
+			"FI_EFA_TX_MIN_CREDITS was set to a value <= 0. This will cause EFA communication to deadlock. Aborting ...\n");
+		abort();
```

Comment:
Try to avoid aborting and uncontrolled output from library whenever possible. How about using FI_WARN to give a warning message and setting the variable to a sane default value?